### ISSUE TITLE
Paste of Images (from Clipboard) no longer works

### ISSUE THREAD

**Issue by richie5um** [laurent22/joplin#5704]
Love Joplin, thank you!

In the Mac version, since I updated to v2.5, I can no longer paste an image from the clipboard. I use this all the time - use Mac to screenshot something onto clipboard, then paste into Joplin.

As a workaround I'm saving the screenshot to disk, and then dragging/dropping the file into Joplin. It works, but isn't as seamless as I used to do.

## Environment

Joplin version: 2.5.12
Platform: Mac
OS specifics: Version 11.6.1

## Steps to reproduce

1. Use Mac to take any screenshot to clipboard - Shift+Command+4
2. Paste into Joplin
3. Image is not pasted.

## Describe what you expected to happen
Markdown image link to be added to Joplin - as happened prior to v2.5.

## Logfile
![CleanShot 2021-11-10 at 12 23 38](https://user-images.githubusercontent.com/907033/141112557-515b296c-668f-4b56-b2c5-8fc83a779e6a.png)

**Comment by richie5um** [comment]
Hi. Thanks. There are no further messages in the console. That was everything.

I've disabled all plugins, restarted, and still not working (for me).
![CleanShot 2021-11-10 at 13 11 25](https://user-images.githubusercontent.com/907033/141119547-20f06d2b-4f40-46ce-a50a-916fbfd95167.png)

**Comment by nikp79** [comment]
I can confirm the same issue on macOS Catalina 10.15.7 on Joplin 2.5.12.  No new plugins were added/enabled for quite some time now.  Tried to also disable plugins to test and still cannot paste anything in

**Comment by laurent22** [comment]
I still can't replicate in either 2.5 or pre-release 2.6 on macOS with or without plugins.

**Comment by nikp79** [comment]
Not sure how else to help test this.  I do use a clipboard manager (Clipy).  I disabled it and still the same issue with 2.5.X.  If there is anything I can do to text, please let me know.  I appreciate everything you've done with Joplin and will do what I can to help!

**Comment by nikp79** [comment]
Updated to 2.6.10 today and still same issue with copy/paste.  However I did a clean uninstall (used [AppCleaner](http://freemacsoft.net/appcleaner/) + manually deleted "~/.config/joplin-desktop"), reinstalled and copy/paste is working again.  Did not think about doing this until today for some reason but glad is working again.  Thanks again for your hard work!

**Comment by nikp79** [comment]
Updated to 2.6.10 today and still same copy/paste issue.  I tried a clean uninstall (including the .config folder), and reinstall with the same issue.  Finally disabled my main third-party screenshot tool (Annotate, which has been outdated beyond my knowledge at the time), used other third-party screenshot tools (Skitch, Greenshot) and also the macOS built-in tools, and those are pasting in correctly in 2..5.x and above.  I believe anyone else having the same issue might want to try switching their third-party screenshot tool first before attempting a clean reinstall.

**Comment by richie5um** [comment]
Great find! I can confirm I get the same behaviour - with a 3rd party screenshot tool (CleanshotX) it won't paste into Joplin. With the built-in Mac screenshot tool, it works. Looks like the pasteboard differs slightly between the two...

### With Mac built-in screenshot tool
![CleanShot 2021-12-21 at 19 20 08](https://user-images.githubusercontent.com/907033/146986021-100b0ad4-617f-463f-95ea-82f47ec21907.png)


### With 3rd party screenshot tool
![CleanShot 2021-12-21 at 19 19 02](https://user-images.githubusercontent.com/907033/146986077-b08eeb8f-d917-4f9e-9144-b308701e0799.png)

**Comment by nikp79** [comment]
Excellent!  I guess this one is solved!

**Comment by kottn** [comment]
I am facing a similar problem with Windows 11.
If I use the Snipping tool (e.g. Win+Shift+S), I can paste without any problem. However, images copied from the Web or Word into the clipboard cannot be pasted.
Does anyone have a solution?

Here is my joplin:
Joplin 2.13.15 (prod, win32)

**Comment by Gitomu** [comment]
Same thing is happening on my Windows and Linux OS based machines. I disabled all plugins and made sure it was fully updated to May 2024 and it is still happening. It's sad this app is quite limited for what I need. I love its feature to allow me syncing between multiple platforms but not being able to see the screenshots I was able to capture during labs repros which scenarios won't be easily reproducible again is a deal breaker for me. 

I'm going back to Obsidian. It may not be the best option, but it surely works for what I need.
